cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDIndividual;
+import uk.gov.hmcts.cmc.domain.models.party.Individual;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+@Component
+public class IndividualMapper implements Mapper<CCDIndividual, Individual> {
+
+    private final AddressMapper addressMapper;
+    private final RepresentativeMapper representativeMapper;
+
+    @Autowired
+    public IndividualMapper(final AddressMapper addressMapper, final RepresentativeMapper representativeMapper) {
+        this.addressMapper = addressMapper;
+        this.representativeMapper = representativeMapper;
+    }
+
+    @Override
+    public CCDIndividual to(Individual individual) {
+
+        final CCDIndividual.CCDIndividualBuilder builder = CCDIndividual.builder();
+        individual.getTitle().ifPresent(title -> builder.title(title));
+        individual.getMobilePhone().ifPresent(mobilePhone -> builder.mobilePhone(mobilePhone));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1612)